tools/tidb-binlog-kafka.md

```diff
@@ -202,8 +204,10 @@ This example describes how to use Pump/Drainer.
     # the path of storing Pump data
     data-dir = "data.pump"
 
-    # the ZooKeeper address; You can set the option to get the Kafka address from ZooKeeper
+    # the ZooKeeper address; You can set the option to get the Kafka address from ZooKeeper; if the namespace is configured in Kafka, you need to keep same configuration here
```

Comment:
if the namespace is configured in Kafka, you need to keep **the** same configuration here